src/MongoDB.Driver.Core/Core/Operations/EndTransactionOperation.cs

```diff
@@ -154,6 +161,17 @@ public CommitTransactionOperation(WriteConcern writeConcern)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AbortTransactionOperation"/> class.
+        /// </summary>
+        /// <param name="recoveryToken">The recovery token.</param>
+        /// <param name="writeConcern">The write concern.</param>
+        public CommitTransactionOperation(BsonDocument recoveryToken, WriteConcern writeConcern)
+            : base(writeConcern)
```

Comment:
In the Java driver we have treated this as a public API since it's documented as such, and therefore added new constructors and deprecated the old ones.  In the next major release we can kill off the old constructors.

Reply:
Acknowledged. I've done the search: the original constructor (prior to this PR) was used three times. Of those two, twice were in tests. I've changed all 3 to use the new constructor.